Open_Foam/Post_Process.ipynb

```diff
@@ -4,129 +4,112 @@
+    "time step continuity errors : sum local = 1.56155e-13, global = 4.42665e-14, cumulative = -5.55876e-09\n",
+    "ExecutionTime = 1042.76 s  ClockTime = 1053 s\n",
     "\n",
-    "Below is the recommended general structure to develop the Post_Process file for the flow of interest. Please note that we are always open to new ideas and modifying the structure below. Thus, do not limit yourself to the following structure and feel free to implement your suggestion and changes. We can modify this structure based on new feedback.\n",
+    "forceCoeffs forcesCoeffs output:\n",
+    "    Cm    = -0.0960072\n",
+    "    Cd    = 1.30985\n",
+    "    Cl    = -0.954175\n",
+    "    Cl(f) = -0.573095\n",
+    "    Cl(r) = -0.38108\n",
     "\n",
-    "#### 1- Start with visualization of the residuals for most important variables within the flow field (e.g. continuity, streamwise velocity and etc.). Discuss how user should examine the residuals trends to comment on accuracy and convergence of the flow field's simulation. \n",
-    "#### 2- Explain how to visualize the contours of important normalized flow field variables (e.g. normalized velocity, pressure and etc.) on one or multiple sections of the CFD domain. Present figure(s) of visualized variable contours. \n",
-    "#### 3- Present a detail discussion on details of the presented results and link your discussion to the previously explained physics of the flow field and expected results in Physics section.\n",
-    "#### 4- Validate your simulation results via presenting a detail comparison between CFD simulation results and corresponding experimental or other numerical results. Please have in mind that the goal here is not to sell your simulation results. The goal is to highlight the limitations and capabilities of the CFD simulations and used CFD package and code.\n",
-    "#### 5- Conclude the post-processing and validation with a last conclusive paragraph highlighting the pros and cons of the CFD simulations.\n",
-    "#### 6- Cite the source(s) that you have used for writing the content of your Domain file considering the copy right rules.  Make sure to cite the sources you use or refer to within the body of the text as well as in the end!\n",
+    "Courant Number mean: 0.0238165 max: 0.565953\n",
+    "```\n",
     "\n",
-    "# Handy Commands for Writing the Physics file: \n",
+    "The residual value of each variable denotes how the numerical value of that variable is differing from the previous time step. Lower value of residual indicates the obtained numerical value of the variable of interest has converged to a specific value. In this simulation this is true for `U_x`, `U_y` and `p` that have residual values by order of at least `1e-6`. The reported `continuity errors` defines how the obtained relevant variables are satisfying the continuity equation. The global continuity error with order of `1e-13` in this problem confirms that continuity equation is being satisfied.\n",
     "\n",
```

Comment:
Do you have history of residuals for a longer run of time steps? If yes, please include it. Furthermore, if time allows it will be great to add some discussion on observation of major spikes in the residual history due to the change of time step.